_includes/manuals/nightly/3.6_upgrade.md

```diff
@@ -169,7 +169,21 @@ And on Debian/Ubuntu installations with Passenger, run:
 
     service apache2 restart
 
-#### Optional Step 5 - Cleanup of RPMs and repositories
+#### Optional Step 5 - Reclaim database space
+
+After database migrations, some space can sometimes be reclaimed. It's a good idea to perform a *full* database vacuum for PostgreSQL rather than relying on the autovacuum feature to claim maximum space possible. For PostgreSQL do the following:
+
+    foreman-maintain service stop --exclude postgresql
+    su - postgres -c 'vacuumdb --full --dbname=foreman'
+    foreman-maintain service start
+
+MariaDB/MySQL users should do the similar:
+
+    foreman-maintain service stop --exclude mysql
+    mysqlcheck --optimize --all-databases
+    foreman-maintain service start
```

Comment:
I think `foreman-maintain` is not available on DEB based installs, so it would be nice to rephrase that, although I'm not 100% sure, what would be corect then.

Reply:
Done, it is also not available on gem installs.